frontend/src/app/document/document-popup/document-popup.component.html

```diff
@@ -1,9 +1,12 @@
 <ng-container *ngIf="document">
     <p-dialog [(visible)]="visible" width="100%"
-        [responsive]="true" [maximizable]="true" [dismissableMask]="true" [draggable]="true" [resizable]="false" [blockScroll]="true"
-        header="Document {{document.position}} of {{page.total}}">
+        [responsive]="true" [maximizable]="true" [dismissableMask]="true" [draggable]="true" [resizable]="false" [blockScroll]="true">
+        <ng-template pTemplate="header">
+            <span>Document {{document.position}} of {{page.total}}</span>
+            <span *ngIf="showNEROption" iaBalloon="Show named entities for this document">NER<ia-toggle (toggled)="toggleNER($event)"></ia-toggle></span>
```

Comment:
Two comments here:

- The label "NER" is not clear. The tooltip text is clearer, but I expect most users would still not know what NER stands for, because they're not familiar with the abbreviation. I would recommend against using this abbreviation anywhere in I-analyzer, since it's not common knowledge. In this case, "show named entities" would be a clearer label.  (NB: if the input has to use the shortest possible label due to a lack of space, then this may not be the right location for it.)
- Currently, NER can be toggled from here in the document popup, but there is no switch in the standalone page for a document. I don't think there is a reason why NER could not be available in that view?